core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -51,37 +51,37 @@ object EntityComponent {
   val allAttributeValuesColumnSize = 65534
 }
 
+import slick.jdbc.MySQLProfile.api._
```

Comment:
This chunk of code was pulled out of the EntityComponent trait below because I needed to reference it outside the trait. Unfortunately this means that I had to import MySQLProfile specifically because the usual way of `import driver.api._` requires `driver` to be in scope which ain't no more.